hoprd/rest-api/src/channels.rs

```diff
@@ -452,6 +452,84 @@ pub(super) async fn close_channel(
     }
 }
 
+#[serde_as]
+#[derive(Debug, Clone, Serialize, utoipa::ToSchema)]
+#[schema(example = json!({
+        "receipt": "0xd77da7c1821249e663dead1464d185c03223d9663a06bc1d46ed0ad449a07118",
+        "channelStatus": "PendingToClose"
+    }))]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct CloseMultipleResponse {
+    /// Receipt for the channel close transaction.
+    #[serde_as(as = "DisplayFromStr")]
+    #[schema(value_type = String)]
+    receipt: Hash,
+}
+
+#[serde_as]
+#[derive(Debug, Clone, Deserialize, utoipa::ToSchema)]
+#[schema(example = json!({
+        "direction": "outgoing",
+        "status": "Open"
+    }))]
+pub(crate) struct CloseMultipleBodyRequest {
+    /// Direction of the channels to close.
+    #[serde_as(as = "DisplayFromStr")]
+    #[schema(value_type = String)]
+    direction: hopr_lib::ChannelDirection,
+
```

Comment:
@mjadach-iv want to put checkboxes next to channels to allow to select which channels to close. 

This API does not seem to allow selective channel closure.